code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -506,7 +506,7 @@
 
 	src.set_light(4,-20)
 
-	CALLBACK(src, /atom/.proc/set_light, 0)
+	addtimer(CALLBACK(src, /atom/.proc/set_light, 0))
```

Comment:
Invalid timer: `time` cannot be null.

Reply:
literally just copy your old timer call and replace the callback with the new one